tests/installation/bootloader_svirt.pm

```diff
@@ -139,6 +139,7 @@ sub run {
         if (my $full_hdd = get_var('HDD_' . $n)) {
             my $hdd     = basename($full_hdd);
             my $hddpath = search_image_on_svirt_host($svirt, $hdd, $hdddir);
+            $size_i = get_var("HDDSIZEGB_$n") if get_var("HDDSIZEGB_$n");
```

Comment:
why not put this in L137
`my $size_i = get_var("HDDSIZEGB_$n") ? 
                      get_var("HDDSIZEGB_$n") : 
                      get_var('HDDSIZEGB', '10');`

or something like that...

Reply:
If `HDDSIZEGB_$n` is not set, it is better to keep the default image size, as we most likely have larger images than 10G. And in such a case the parameter would be ignored because the image won't be bootable if the backing file is smaller than the original image.
And I prefer to keep mapping `HDD_$n` to `HDDSIZEGB_$n`.

Reply:
Still i think it makes sense to have the assignment in one place. And you could work out the default case.
Something like that could be `my $size_i = get_var("HDDSIZEGB_$n") ? get_var("HDDSIZEGB_$n") : get_var('HDDSIZEGB') // 10;`
but it s up to you how would you like to implement it. 
The point is that the code is easier readable and it is easy-made change. wdyt?